tools/settings.py

```diff
@@ -102,12 +103,15 @@ def read_js_settings(filename, attrs):
       with open(filename) as fh:
         settings = fh.read().replace('//', '#')
       settings = re.sub(r'var ([\w\d]+)', r'attrs["\1"]', settings)
+      settings = settings.replace('= false;', '= False')
+      settings = settings.replace('= true;', '= True')
```

Comment:
This could be brittle?

Maybe at minimum `=\s*false\s*` -> `= False`? Or using word boundaries: `\bfalse\b` -> `False`?

I know you expect the file to have a consistent style, but getting probably a python parsing error exception(?) because of unexpected whitespace, e.g.  `=false;` can build up that kind of a "fragile beast to operate" type of feeling of flakiness, e.g. if one is just testing changes.

(also is the semicolon swallowing intentional?)

Reply:
Agree